jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -1,104 +1,121 @@
-    3. Enablement - This determines if a policy violation should be enabled.
+After declaring the `org` package and `policy_name` rule, policies can then be defined as a list of rules. Each rule is composed of three parts:
+
+* Evaluation - Evaluates whether the config contains the policy violation
+* Enforcement status - Determines how a violation should be enforced.
+* Enablement - Determines if a policy violation should be enabled.
 
 Using this format allows policy writers to create custom helper functions without impacting CircleCI's ability to
-parse policy evaulation output. Policies all have access to config data through the `input` variable. The `input` is the project config being evaluated. Since the `input` matches the CircleCI config, you can write rules to enforce a desired state on any any available config strucutre i.e. `jobs`, `workflows`, etc.
+parse policy evaulation output. Policies all have access to config data through the `input` variable. The `input` is the project config being evaluated. Since the `input` matches the CircleCI config, you can write rules to enforce a desired state on any available config element, for example, `jobs` or `workflows`.
 
-```shell
+[source,rego]
+----
 input.workflows     # an array of nested structures mirroring workflows in the CircleCI config
 input.jobs          # an array of nested structures mirroring jobs in the CircleCI config
-```
+----
 
+==== Define a rule
 
-### Rule Definition
-{: #rule-definition }
-#TODO: update "evaluation name" to "rule definition"
+#TODO: update "evaluation name" to "rule definition"#
 
 In OPA rules can produce any type of output. At CircleCI, rules that produce violations must have outputs of the following types:
-- string
-- string array
-- map of string to string
 
-This is because rule violations must produce error messages that individual contribtors and secops can act upon.
-Helper rules that produce differently typed outputs can still be defined, but rules that will be considered when making CircleCI Decisions must have this output. For more information see Rule Enablement below.
+* String
+* String array
+* Map of string to string
+
+This is because rule violations must produce error messages that individual contributors and SecOps can act upon.
+Helper rules that produce differently typed outputs can still be defined, but rules that will be considered when making CircleCI decisions must have the output types specified above. For more information see the <<#enablement>> section below.
+
+===== Evaluation
 
 This is how the decision engine determines if a config violates the given policy. The evaluation defines the name and id of the rule, checks a condition, and returns a user-friendly string describing the violation. Rule evaluations include the **rule name** and an **optional rule id**. The rule name will be used to enable and set the enforcement level for a rule.
 
-#TODO: specify expected outputs
+#TODO: specify expected outputs#
```

Comment:
@sagar-connect could you check if this "To do" has been addressed?

Reply:
same comment for this point as well, need some examples of violation outputs.